pandas/io/sas/sas.pyx

```diff
@@ -177,17 +177,14 @@ cdef const uint8_t[:] rdc_decompress(int result_length, const uint8_t[:] inbuff)
             rpos += cnt
 
         # short pattern
-        elif (cmd >= 3) & (cmd <= 15):
+        else:
             ofs = cnt + 3
             ofs += <uint16_t>inbuff[ipos] << 4
             ipos += 1
             for k in range(cmd):
                 outbuff[rpos + k] = outbuff[rpos - <int>ofs + k]
             rpos += cmd
 
-        else:
```

Comment:
This case is actually impossible and I removed it so that nobody is confused why we aren't testing the `raise ValueError`.